pkg/test/runner/runner.go

```diff
@@ -275,6 +282,10 @@ func (r *Runner) runFnRender() error {
 		}
 	}
 
+	if r.IsOutOfPlace() {
+		destDir = filepath.Join(pkgPath, outDir)
```

Comment:
The output dir is a subdirectory in the package? I am hesitating is this a good idea.

Reply:
So the only downside to this approach (or pattern) is that your rendered config lives side by side with original source config and that can be painful for `git diff` (review and approval workflows). But that exactly is the use-case `git diff` for the test harness in this case.

I realized we can extend this to support `function results` as well and get away from the special handling of them.